tethys_portal/settings.py

```diff
@@ -220,7 +220,7 @@
 )
 
 default_installed_apps = [
-    "channels",
```

Comment:
channels is not needed?

Reply:
According to the latest documentation, it appears it is not necessary unless using Channel's runworker command (see [here](https://channels.readthedocs.io/en/stable/installation.html)). However, this likely isn't the case on the previous versions of Django. Since including it seems to be optional now, we should probably include it anyway to get backwards compatibility.